assets/pages/new-contributions-landing/components/MustSignIn.jsx

```diff
@@ -16,17 +16,19 @@ type PropTypes = {|
   userTypeFromIdentityResponse: UserTypeFromIdentityResponse,
   contributionType: Contrib,
   checkoutFormHasBeenSubmitted: boolean,
+  email: string,
 |};
 
 
 // ----- Functions ----- //
 
 // Build signout URL from given return URL or current location.
-function buildUrl(returnUrl: ?string): string {
+function buildUrl(returnUrl: ?string, email: string): string {
 
   const encodedReturn = encodeURIComponent(returnUrl || window.location);
+  const encodedEmail = encodeURIComponent(email);
 
-  return `https://profile.${getBaseDomain()}/signin?returnUrl=${encodedReturn}`;
+  return `https://profile.${getBaseDomain()}/signin?returnUrl=${encodedReturn}&email=${encodedEmail}`;
```

Comment:
ah, we actually want `/signin/current` to go straight to the second page of the flow

Reply:
ah nice